src/api/types.ts

```diff
@@ -334,3 +334,14 @@ export enum JobStatus {
   Failure = 'FAILURE',
   TimedOut = 'TIMED_OUT',
 }
+
+export enum ContractType {
+  Production = 'PRODUCTION',
+  Trial = 'TRIAL',
+}
+
+export interface ExpirationResponse {
+  contractType: ContractType;
+  expirationDate?: string;
+  shutdownDate?: string;
```

Comment:
Technically I think these are 
```suggestion
  expirationDate: string | null;
  shutdownDate: string | null;
```
based on how they get encoded in JSON, right? (I know this is probably copy/paste from brain-app but it can't hurt to fix 😸)